### PR TITLE
feat: fetch sdk latest release version

### PR DESCRIPTION
**Pull request by hezean**
Works with FeatureProbe/feature-probe-api#23

### CODE REVIEW

File: src/constants/api/misc.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+let origin = '/api';
+
+const MiscURI = {
+    sdkVersionURI: `${origin}/misc/sdk/:key`,
+}
```
Comment (FengShang) [inline src/constants/api/misc.ts:5]: Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
};
```

File: src/constants/api/misc.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+let origin = '/api';
+
+const MiscURI = {
+    sdkVersionURI: `${origin}/misc/sdk/:key`,
```
Comment (FengShang) [inline src/constants/api/misc.ts:4]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
  sdkVersionURI: `${origin}/misc/sdk/:key`,
```

File: src/services/misc.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import request from '../utils/request';
```
Comment (FengShang) [inline src/services/misc.ts:1]: Codacy has a fix for the issue: [Replace `'../utils/request'` with `"../utils/request"`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
import request from "../utils/request";
```

File: src/services/misc.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import API from '../constants/api';
+
+export const getSdkVersion = async<T> (key: string) => {
+    const url = `${
+        API.sdkVersionURI
+            .replace(':key', key)
+    }`;
+
+    return request<T>(url);
```
Comment (FengShang) [inline src/services/misc.ts:10]: Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
  return request<T>(url);
```

File: src/constants/api/index.ts [hunk 5]
```diff
@@ -2,6 +2,7 @@ import project from './project';
 import toggle from './toggle';
 import user from './user';
 import member from './member';
+import misc from './misc';
```
Comment (FengShang) [inline src/constants/api/index.ts:5]: Codacy has a fix for the issue: [Replace `'./misc'` with `"./misc"`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
import misc from "./misc";
```

File: src/constants/api/misc.ts [hunk 6]
```diff
@@ -0,0 +1,7 @@
+let origin = '/api';
```
Comment (FengShang) [inline src/constants/api/misc.ts:1]: Codacy has a fix for the issue: [Replace `'/api'` with `"/api"`](https://app.codacy.com/gh/FeatureProbe/feature-probe-ui/pullRequest?prid=9905084)
```suggestion
let origin = "/api";
```

**Comment by hezean** [comment]
@frankzhaopeng can you please help review this pr? many thanks

**Comment by frankzhaopeng** [comment]
> @frankzhaopeng can you please help review this pr? many thanks
Hi, I have reviewed ur pr. New features are going to be released next week, some conflicts needs to be resolved after that, We will confirm ur pr next week. Thanks.

**Pull request merged**